app/controllers/events_controller.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+class EventsController < ApplicationController
+  before_action :log_in_user, only: [:create]
+
+  def new
+    if signed_in?
+      @event = current_user.events.build
+    else
+      flash[:danger] = 'Kindly log in to create an event'
+      redirect_to signin_path
```

Comment:
You are getting an error here. Please, see #2.

Reply:
![image](https://user-images.githubusercontent.com/29500109/69880619-5df01200-1305-11ea-8a46-7a40265534cc.png)

Issue was addressed by omitting logic method from "return unless signed_in?" (caused by rubocop -a) to ""return if signed_in?".